__tests__/index.js

```diff
@@ -102,15 +105,17 @@ test.concurrent('should add lockfile package', async () => {
   expectAddSuccessfullOutput(stdout, 'lockfile');
 });
 
-test.concurrent('should add progress package globally', async () => {
-  const stdout = await execCommand('global',
-    ['add', 'progress', '--global-folder', './global'],
-    'run-add-progress-globally',
-    true);
+if (semver.satisfies(ver, '>=5.0.0')) {
```

Comment:
Maybe add a comment to explain why it doesn't run on <5? So that it can be understood without having to find the PR